site/en/guide/estimator.ipynb

```diff
@@ -637,7 +625,7 @@
       "source": [
         "## SavedModels from Estimators\n",
         "\n",
-        "Estimators export SavedModels through [`tf.Estimator.export_saved_model`](https://www.tensorflow.org/api_docs/python/tf/estimator/Estimator#export_saved_model)."
```

Comment:
Same here...

Reply:
Hi, same as in https://github.com/tensorflow/docs/pull/1736#discussion_r518244265 - I thought these get auto-linked to API references during the notebook-to-TensorFlow.org preprocessing